libobs/obs-source.c

```diff
@@ -366,16 +368,17 @@ static obs_source_t *obs_source_create_internal(const char *id,
 }
 
 obs_source_t *obs_source_create(const char *id, const char *name,
-		obs_data_t *settings, obs_data_t *hotkey_data)
+		obs_data_t *settings, obs_data_t *hotkey_data, bool dsk)
```

Comment:
Note: this seems to break obs-ndi. Would this be considered a breaking change to the API?

"'obs_source_create': function does not take 4 arguments"

Reply:
If it changes the API and external things (plugins) rely on that API, and the change causes those plugins to break, then yes, I'd say that's a breaking change.  In semver, such a change would be reserved for a major release like OBS 24.0.0 (or libobs 24.0.0, which is tied to OBS releases).

Reply:
Then I would suggest a backwards compatible helper function that accepts 4 arguments and passes false for the last parameter on its own.